### PR TITLE
configure and create cacheDirectory

### PR DESCRIPTION
**Pull request by staabm**
as discussed in https://github.com/rectorphp/rector/discussions/8357#discussioncomment-8420673

### CODE REVIEW

File: resources/docs/cache-in-ci.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ return RectorConfig::configure()
         cacheClass: FileCacheStorage::class,
 
         // specify a path that works locally as well as on CI job runners
-        cacheDirectory: './var/cache/rector'
+        cacheDirectory: '/tmp/rector'
```
Comment (staabm) [inline resources/docs/cache-in-ci.md:15]: using `/tmp/rector` as it works well for use.

phpstan by default uses `/tmp/phpstan`

File: resources/docs/cache-in-ci.md [hunk 2]
```diff
@@ -38,10 +38,12 @@ On GitHub Actions, you can use the [built-in cache action](https://github.com/ac
       - name: Rector Cache
         uses: actions/cache@v3
         with:
-          path: ./var/cache/rector
+          path: /tmp/rector
           key: ${{ runner.os }}-rector-${{ github.run_id }}
           restore-keys: ${{ runner.os }}-rector-
 
+      - run: mkdir -p /tmp/rector
```
Comment (samsonasik) [inline resources/docs/cache-in-ci.md:45]: should it run before line 38 above executed?

Comment (staabm) [reply resources/docs/cache-in-ci.md:45]: Nope. Needs to be here.

**Comment by samsonasik** [comment]
Ok, thank you @staabm

**Pull request merged**